### ISSUE TITLE
make donator job changes (to encourage donating)

### ISSUE THREAD

**Issue by AdamElTablawy** [yogstation13/yogstation#605]
add a few donor jobs to encourage donating that don't have really anything speshul
1. coroner

starts with-

1x forensic scanner

1x latex gloves

1x sterile mask

1x operating surgeon drape thing

surgical tools and operating bed 

office by morgue near paramedic office/paramedics share office with coroner

coroner ID has-morgue, medical, med storage, paramedic access

job desc-mostly RP, when corpses come into the morgue checks them for cause of death with health analyzer and forensic scanner, can preform surgery on corpses with no head ect to reattach them for cloning, ect

blueshield-

go look on literally any server that has a blueshield role for a description of what they do.

Judge/Magistrate-

starts with 1x powdered wig

1x gavel

1x robes

1x spacelaw

ID-brig, courtroom, brig timers

job desc-mostly RP/fluff, serves as lawyer buttbuddies in that they have the final say in space law if something is according to space law or not

for example HoS says to give clown 10 minutes for assorted crimes, judge rules that is the incorrect time, sec officers obey him in ruling

can't override captain's execution orders but CAN determine if captain ordered someone executed if the person who commited the crime is actually capital-crime and thus execution worthy or if captain's just wrong

also is the judge in the courtroom if a trial happens (rarely). can't be antaggo.

Waiter-

starts with 1x all of chef equipment but kitchen cape is replaced with waistcoat and chef hat is replaced with top hat

ID-kitchen, bar

chef buttbuddy. chef makes the food, waiter delivers it. that's all. 

as you can see these jobs are all very boring aside from judge and blueshield but hey people will pay for stupid shit so this will probably encourage more donating thanks

**Comment by utahclock** [comment]
we used to have waiter its currently commented out.

**Comment by EMT321** [comment]
If there were judges I would hope we would see more trials.

**Comment by aksuduud** [comment]
If sec is enforced by rules to follow the judges ruling then the judge role has sickening grief/asshole potential, I hope admins jobban swiftly for abuse. Trials don't fit the current meta and they are just a waste of time for everyone involved.

**Comment by Alek2ander** [comment]
You've got to be kidding me

**Comment by AdamElTablawy** [comment]
@alek2ander

I hope you are talking to aksu and not me.

I know the suggestion is half assed but still. 

As for aksu-

The judge determines if a sentencing is lawful or not-so that if the captain calls something a capital crime and orders said person executed, they can override him and name the proper sentencing for the crime.

But he himself cannot assign sentencing/decide if they should be harsher/more lenient/order execution, ect.

Essentially, the judge exists completely outside of the chain of command-a walking, talking space law, who exists to correct people who misassign sentences to prisoners (he walked into the bridge, capital crime, execute-judge denies.) But cannot say, assign someone for execution the captain ordered perma or otherwise.

He exists to correct incorrect sentences, name the crimes of the convicted, and assign the proper time or state they are a capital criminal-actual enforcement falls upon the warden.

Basically he's a half hard-counter to shitcurity who give wrong times, ect. The lawyers exist to determine if the client is innocent or guilty or deserve a reduced sentence-the judge determines if this is true, and the Captain makes the final call.

Make sense?

**Comment by Alek2ander** [comment]
I'm talking to both. Remember we had the recovery agent and how most people hated it? This is just as bad.

**Comment by EMT321** [comment]
I dont think any special snowflake roles should get guns. Additionally, server is currently financially secure for the next 4 months. This might be a bit much.

**Comment by AdamElTablawy** [comment]
@Alek2Ander

It's not AS bad as recovery agent, but...

Please highlight what needs to be changed.
